sbpy/data/names.py

```diff
@@ -167,15 +166,15 @@ def parse_comet(s):
         +--------------------------------+-------+------+-------+------------+----------------------------+
 
         """
-        
+
         import re
 
         # define comet matching pattern
         pat = ('^(([1-9][0-9]*[PDCXAI]'
                '(-[A-Z]{1,2})?)|[PDCXAI]/)' # typ/number/fragm [0,1,2]
-               '|([-]?[0-9]{3,4}[ _][A-Z]{1,2}[0-9]{1,3}(-[1-9A-Z]{0,2})?)' 
+               '|([-]?[0-9]{3,4}[ _][A-Z]{1,2}[0-9]{1,3}(-[1-9A-Z]{0,2})?)'
                # designation [3,4]
-               '|(([A-Z][a-z]?[A-Z]*[a-z]*[ -]?[A-Z]?[1-9]*[a-z]*)' 
+               '|(([dA-Z][a-z\']? ?[A-Z]*[a-z]*[ -]?[A-Z]?[1-9]*[a-z]*)'
```

Comment:
Also need to support a leading lower-case 'v' for  271P/van Houten-Lemmon (according to Wikipedia).

@migueldvb , @mommermi But this makes me think, do we really want to enforce Proper Capitalization?

Reply:
Thanks, I added a lower case v in the character set used to match the first letter of the comet name. I'm not sure if we need to ensure the right capitalization of the name. This pattern will allow several different choices of case for some names.